src/components/Select/Select.tsx

```diff
@@ -37,6 +37,9 @@ import { useMaxVisibleSections } from '../../hooks/useMaxVisibleSections';
 
 import styles from './select.module.scss';
 
+const inputPaddingLeft: number = +styles.inputPaddingLeft;
```

Comment:
is all this RTL compatible or is it going to make it more challenging? asking for the sake of knowledge. Not a request for action

Reply:
I'll rename, we may toggle the dir in the computation.